app/submitter/convert_payload_0_0_1.py

```diff
@@ -7,61 +7,68 @@
 
 # pylint: disable=too-many-locals
 def convert_answers_to_payload_0_0_1(
-    metadata, answer_store, list_store, schema, routing_path
+    metadata, answer_store, list_store, schema, full_routing_path
 ):
     """
     Convert answers into the data format below
+    Answers for list items are not currently supported
     'data': {
           '001': '01-01-2016',
           '002': '30-03-2016'
         }
```

Comment:
Not sure about this comment, or its position. I assume your are talking about list_items in connection to relationships, as list items could be deemed as a list of items like a checkbox ["Eggs", "Bacon", "Sausages"]

Reply:
Well, not specifically relationships (although relationship answer data isn't used in Business surveys). This was referring to any repeating section answers for a given list_item_id. I'll update this comment to `list_item_id` for clarification. The comment itself is intended to be visible at the top of the docstring.